common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -109,6 +110,13 @@ function (Iterator) {
             speedsContainer.html(speedsList.join(''));
             this.speedLinks = new Iterator(speedsContainer.find('.speed-option'));
             this.state.el.find('.secondary-controls').prepend(this.el);
+            if (this.state.speed) {
+                initialSpeed = this.state.speed;
+            } else {
+                // default if no speed is set
+                initialSpeed = "1.0";
```

Comment:
If we were to use `speeds` we'd still want to find "1.0" in that object, so in a sense it would still be hard-coded. I'd like the default to be "normal" speed (1.0x). What would you suggest here?